flatpak-proxy.c

```diff
@@ -127,9 +127,12 @@
  * reply_serials, i.e. that a reply can only be sent once and by the real
  * recipient of an previously sent method call.
  *
- * We don't however trust the serials from the client. We verify that
- * they are strictly increasing to make sure the code is not confused
- * by serials being reused.
+ * The serials are separated into namespaces of lower_bit=0 for client
```

Comment:
The working is a bit odd here. I would not call this 'namespaces'. Suggestion: "Odd serial numbers are reserved for messages created by the proxy itself."

Reply:
Fixed.